### PR TITLE
Updated sentence for better readability.

### PR DESCRIPTION
**Pull request by lukemurraynz**
Updated sentence for better readability. #ATCP

### CODE REVIEW

File: Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Before you migrate mailboxes to Microsoft 365 or Office 365 by using a cutover m
 
    - [How to configure Outlook Anywhere with Exchange 2003](/previous-versions/office/exchange-server-2007/aa996922(v=exchg.80))
 
-2. You must use a certificate issued by a trusted certification authority (CA) with your Outlook Anywhere configuration in order for Microsoft 365 or Office 365 to run a cutover migration. For cutover migration you will to add the Outlook Anywhere and Autodiscover services to your certificate. For instructions on how to set up certificates, see:
+2. You must use a certificate issued by a trusted certification authority (CA) with your Outlook Anywhere configuration in order for Microsoft 365 or Office 365 to run a cutover migration. If you are doing a cutover migration, you will to add the Outlook Anywhere and Autodiscover services to your certificate. For instructions on how to set up certificates, see:
```
Comment (johnynitro) [inline Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md:105]: If you are doing a cutover migration, you will **need** to add

Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md:105]:
```suggestion
2. You must use a certificate issued by a trusted certification authority (CA) with your Outlook Anywhere configuration in order for Microsoft 365 or Office 365 to run a cutover migration. If you are doing a cutover migration, you will need to add the Outlook Anywhere and Autodiscover services to your certificate. For instructions on how to set up certificates, see:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lukemurraynz, kindly commit the suggested change. Many thanks.

File: Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md (hunk 1 above)
Comment (lukemurraynz) [reply Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md]: Updated.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**